x-pack/plugins/endpoint/common/models/event.ts

```diff
@@ -46,3 +47,42 @@ export function parentEntityId(event: ResolverEvent): string | undefined {
   }
   return event.process.parent?.entity_id;
 }
+
+export function eventCategoryDisplayName(event: ResolverEvent): EventCategory {
+  const eventTypeToNameMap = new Map<string, EventCategory | undefined>([
+    ['process', 'Process'],
+    ['alert', 'Alert'],
+    ['security', 'Security'],
+    ['file', 'File'],
+    ['network', 'Network'],
+    ['registry', 'Registry'],
+    ['dns', 'DNS'],
+    ['clr', 'CLR'],
+    ['image_load', 'Image Load'],
+    ['powershell', 'Powershell'],
+    ['wmi', 'WMI'],
+    ['api', 'API'],
+    ['user', 'User'],
+  ]);
+
+  // Returning "Process" as a catch-all here because it seems pretty general
+  let eventCategoryToReturn: EventCategory = 'Process';
```

Comment:
should this just be left as `Event` because its more of a catchall than process?

Reply:
That's not one of the things it can _be_ - at least according to the understanding presented by design in the mocks. Maybe we could make that an extra entry as a catch-all? Seems it would make sense, I just didn't want to take liberties with the mocks.

Reply:
Yeah that could be a good option, I was just wondering if it'd be possible for some event to come back and fall under that catch-all that didn't really fit the term `Process`

Reply:
Looking at the list of _possible_ things above, do you see one that fits better than `Process`?

Reply:
I would argue that if it didn't fit one of those preset things, maybe it could go into some `Unknown` aggregate? That way `Process` isn't filled up with random events that might not really fit with what the user would typically see as process events. Thoughts?